packages/fast-components-styles-msft/src/text-field/index.ts

```diff
@@ -0,0 +1,34 @@
+import { IDesignSystem } from "../design-system";
+import { ComponentStyles, ICSSRules } from "@microsoft/fast-jss-manager";
+import { ITextFieldClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import { toPx } from "@microsoft/fast-jss-utilities";
+import * as Chroma from "chroma-js";
+
+const styles: ComponentStyles<ITextFieldClassNameContract, IDesignSystem> = {
+    textField: {
+        color: "#757575",
```

Comment:
You can use chroma to mix colors to get this result. See button for examples.

Reply:
This would go for all hardcoded hex values.

Reply:
Could this be moved into a utility? Seems like its going to be used frequently.